src/util/threadpool.rs

```diff
@@ -445,175 +451,94 @@ mod test {
         let mut task_pool = ThreadPool::new(name, concurrency, BigGroupThrottledQueue::new(1));
         let (jtx, jrx) = channel();
         let group_with_big_task = 1001 as u64;
-        let sleep_duration = Duration::from_millis(50);
         let recv_timeout_duration = Duration::from_secs(2);
```

Comment:
It's fine to use the name `timeout`, since it's the only duration in this test case.